app/src/org/commcare/android/framework/EntitySubnodeDetailFragment.java

```diff
@@ -69,6 +71,17 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle sa
         return rootView;
     }
 
+    private EvaluationContext prepareEvaluationContext(TreeReference childReference) {

```

Comment:
I don't have the background in this area of the code to understand the PR's description, but I imagine adding a comment here that says essentially the same thing would be helpful for readers that do.